src/markdown-pages/contribute-to-quickstarts/instrument-library/send-events.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Send Events from Your Library'
+template: 'GuideTemplate'
+description: 'With our Events API you can ingest your data as the first step towards your quickstart contribution.'
+tags:
+  - quickstart
+---
+
+<Intro>
+
+This guide will walk you through how to instrument your application to send event data into the New Relic's [Event API](https://docs.newrelic.com/docs/data-apis/ingest-apis/introduction-event-api/). The Event API allow you to send events to New Relic without a need for an agent. First, let's cover what events are. An event is a record of something happening somewhere. What triggers the event and what data is captured is up to you to customize, which we'll discuss next. 
+
+</Intro>
+
+## Capture Events in your Library
+
+At New Relic, we report events to data objects also called events. These events have multiple attributes (key-value pairs). We name each event record in our system with the event-type key. 
+```json
+[{"event-type": "transaction"}]
+```
+The event-type key is how you will query the event data from NRQL. using the above example, this is how we would query for the data in NRQL:
+
+```sql
+From transaction select *
+```
+The rest in the event can be any custom key-value data, based on your library. When preparing the payload, consider the use case that makes the most sense for your users, and build the events accordingly. Here are some more resources about event data if you need more information.
+-  [Events at New Relic](https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types/#events-new-relic)
+-  [Formatting JSON](https://docs.newrelic.com/docs/data-apis/ingest-apis/introduction-event-api#instrument)
+
+## Send your events to New Relic
+There are a few ways we will discuss in order to send your event data into New Relic. We have our [telemetry SDK's](https://docs.newrelic.com/docs/data-apis/ingest-apis/telemetry-sdks-report-custom-telemetry-data), which are development kits instrumented to help you easily set up a POST request via several very common languages (C, Java, Python, .NET, etc.). We also have a basic curl command, which can be used on a windows or linux system to make a POST request to New Relic. We also have a prometheus option, if you use prometheus and have difficulty with the first two options.
```

Comment:
I have a few comments about this paragraph:

- Can you order these methods the same order that you use for the sections?
- Can you define SDK before using the acronym?
- Can you limit the languages you refer to to the ones you provide examples for?
- Can you make this into a list instead of a paragraph?

Reply:
This was updated with the recent commits, but I'll leave this open. until it's reviewed.